manage.py

```diff
@@ -562,25 +563,84 @@ def _run_field_lists(name, table, tables_only):
         no_field_list_type = sql.SQL('TABLE' if table.is_table else 'VIEW')
         final_summary_type = sql.SQL('VIEW')
 
+    counts_per_path_select = """
+        SELECT
+            {primary_keys},
+            path,
+            GREATEST(sum(array_item), sum(object_property)) path_count
+        FROM
+            {summary_table}
+        CROSS JOIN
+            flatten({summary_table}.{data_column})
+        GROUP BY
+            {primary_keys}, path
+    """
+
+    # for contracts and award add paths across the awardID join.
+    if summary_table in ('contracts_summary', 'awards_summary'):
+        if summary_table == 'contracts_summary':
+            variables.extend(['awards/', 'awards'])
+        else:
+            variables.extend(['contracts/', 'contract'])
+
+        counts_per_path_select += """
+        UNION ALL
+
+        SELECT
+            {qualified_primary_keys}, %s || path, GREATEST(sum(array_item), sum(object_property)) path_count
```

Comment:
Why not this, to keep it to one variable above. Also split across lines, for easier comparison to above SQL.

```suggestion
            {qualified_primary_keys},
            %s || '/' || path,
            GREATEST(sum(array_item), sum(object_property)) path_count
```